modules/navier_stokes/doc/content/source/fvkernels/WCNSFV2PInterfaceAreaSourceSink.md

```diff
@@ -0,0 +1,114 @@
+- $d_p$ is the best estimate for the dispersed phase particle diameter,
+- $h^{b \rightarrow d}$ is the mass exchange coefficient from the bulk to the dispersed phase,
+
+The cutoff volumetric fraction $\alpha_d^{co}$ is included because the mass transfer term
+in Hibiki and Ishii's model is not physical for low volumetric fractions of the dispersed phase.
+Below the cutoff limit, the dispersed phase is modeled as spherical particles.
+
+!alert note
+The user should select the cutoff volumetric fraction $\alpha_d^{co}$ as the limit at which
+the modeled flows transition away from bubbly flow, i.e., below the cutoff limit there is an
+implicit assumption that the flow behaves as a bubbly flow.
+
+
+## Interface area concentration source due to coalescence
+
+The reduction in interface area concentration due to coalescence of the dispersed phase is modeled as follows:
+
+\begin{equation}
+S_C = -\left( \frac{\alpha_d}{\xi_p} \right)^2 \frac{\Gamma_C \alpha_d^2 u_{\epsilon}}{\tilde{d_p}^{11/3} (\alpha_d^{max}- \alpha_d)}
+\operatorname{exp} \left( -K_C \frac{\tilde{d_p}^{5/6} \rho_b^{1/2} u_{\epsilon}^{1/3}}{\sigma^{1/2}} \right)\,,
+\end{equation}
+
+where:
+
+- $\alpha_d$ is the volumetric fraction of the dispersed phase, e.g., the void fraction if the dispersed phase is a gas,
+- $u_{\epsilon} = \left( \| \vec{u} \| \ell \| \nabla p \| / \rho_m \right)^{1/3}$ is the friction velocity due to pressure gradients, with $\| \vec{u} \|$ being the norm of the velocity vector, $\ell$ a characteristic mixing length, $\| \nabla p \|$ the norm of the pressure gradient, and $\rho_m$ the mixture density,
+- $\tilde{d_p} = \psi \frac{\alpha_d}{\xi_p}$ is the model estimate for the dispersed phase particle diameter, with $\psi$ being a shape factor, which is, for example, $\psi=6$ for spherical particles,
+- $\alpha_d^{max}$ is the maximum volumetric fraction admitted by the model, in absence of data we recommend taking $\alpha_d^{max}=1$,
+- $\rho_b$ is the bulk phase density, e.g., for air bubbles in a water flow it would be the density of water,
+- $\sigma$ is the surface tension between the two phases,
+- $\Gamma_C = 0.188$ and $K_C = 0.129$ are closure coefficients of the model.
+
+!alert note
+Many of the parameters in the coalescence model are provided as functors,
+which means that spatially dependent fields may be specified for these parameters.
+However, the validation of this model has only been performed using constant parameters for this model.
+
+## Interface area concentration source due to breakage
+
+The increase in interface area concentration due to breakage of the dispersed phase is modeled as follows:
+
+\begin{equation}
+S_B = \left( \frac{\alpha_d}{\xi_p} \right)^2 \frac{\Gamma_B \alpha_d (1 - \alpha_d) u_{\epsilon}}{\tilde{d_p}^{11/3} (\alpha_d^{max}- \alpha_d)}
+\operatorname{exp} \left( -K_B \frac{\sigma}{\rho_b \tilde{d_p}^{5/6} u_{\epsilon}^{2/3}} \right)\,,
+\end{equation}
+
```

Comment:
Inside the exponential term, it should be $\tilde{d_p}^{5/3}$ instead of $\tilde{d_p}^{5/6}$ according to the reference paper. I checked, the code implementation is correct

Reply:
Thanks. Corrected.